libs/windy-sounding/types/client/TopoMap.d.ts

```
@@ -0,0 +1,10 @@
+import { Renderer } from '@windy/Renderer';
+import type { FullRenderParameters } from '@windy/Layer.d';
+import type { Renderers } from '@windy/Renderer.d';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)